client/document/index.jsx

```diff
@@ -68,6 +69,7 @@ class Document extends React.Component {
 			( user ? `var currentUser = ${ jsonStringifyForHtml( user ) };\n` : '' ) +
 			( isSupportSession ? 'var isSupportSession = true;\n' : '' ) +
 			( app ? `var app = ${ jsonStringifyForHtml( app ) };\n` : '' ) +
+			( application ? `var application = ${ jsonStringifyForHtml( application ) };\n` : '' ) +
```

Comment:
Instead of specifying a new var here, would it be better if we use the `app` object above?

Reply:
Good idea :+1: